### ISSUE TITLE
EPIC: Fix issues raised by WCAG 2.0 AA audit

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#1102]
The following issues were found by our external auditor and need to be fixed so that we can be fully WCAG 2.0 AA compliant: 

- [x] Confirm/Fix markup validation errors (https://github.com/hypothesis/client/issues/2059)
- [x] Style links in client tutorial to make them accessible (https://github.com/hypothesis/client/issues/2054)
- [x] Communicate to assistive tech whether sidebar is open or closed (https://github.com/hypothesis/client/issues/2055)
- [x] Incorrect role used with aria-checked (currently being worked on by Kyle)
- [x] Review how accessible labels for buttons are implemented (https://github.com/hypothesis/client/issues/2056)
- [x] "More" button does not serve a purpose for screen reader users and is not accurately described (https://github.com/hypothesis/client/issues/2057)
- [x] Confusing experience for assistive tech & keyboard users when sidebar is closed (https://github.com/hypothesis/client/issues/2058)

There were some usability improvements suggested as part of this audit, and I will write those up separately. The issues needed for compliance take priority and should be addressed before any of the optional usability improvements.

**Comment by klemay** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility audit follow-up](https://trello.com/c/o6lluE7w/14-accessibility-audit-follow-up)

**Comment by lyzadanger** [comment]
All items are complete! May we close?

**Comment by klemay** [comment]
@lyzadanger incredible work from you and the rest of the team! Closing!
